src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -311,6 +314,14 @@ public Task CopyToAsync(Stream stream)
             return CopyToAsync(stream, null);
         }
 
+#if NET46
+        // Workaround for HttpWebRequest synchronous resubmit
```

Comment:
This comment was copied as-is from the original .NET Framework source code.  The code is required because the underlying HttpWebRequest implementation cannot use CopyToAsync and only uses sync based CopyTo.